dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -736,13 +736,41 @@ def test_profile_merge_edge_case(self):
         self.assertEqual(profiler.min, 2.0)
         self.assertEqual(profiler.max, 5.0)
 
+    def test_custom_bin_count_merge(self):
+
+        options = FloatOptions()
+        options.histogram_and_quantiles.method = None
+        options.histogram_and_quantiles.hist_bin_count = 10
+
+        data = [2.0, 'not a float', 6.0, 'not a float']
+        df = pd.Series(data).apply(str)
+        profiler1 = FloatColumn("Float", options)
+        profiler1.update(df)
+
+        data2 = [10.0, 'not a float', 15.0, 'not a float']
+        df2 = pd.Series(data2).apply(str)
+        profiler2 = FloatColumn("Float", options)
+        profiler2.update(df2)
+
+        # no warning should occur
+        with warnings.catch_warnings(record=True) as w:
+            profiler1 + profiler2
+        self.assertListEqual([], w)
+
+        # make bin counts different and get warning
+        profiler2.user_set_histogram_bin = 120
+        with self.assertWarnsRegex(UserWarning,
+                                   'User set histogram bin counts did not '
+                                   'match. Choosing the larger bin count.'):
+            merged_profile = profiler1 + profiler2
+        self.assertEqual(120, merged_profile.user_set_histogram_bin)
```

Comment:
Would a user ever do it this way or is the correct usage to set this via options?

Reply:
the intent is not that a user would do this, the intent is to not need to create options every time to get a test to be functional for eval